### PR TITLE
Recent transcriptions

### PR DESCRIPTION
**Pull request by rafelafrance**

### CODE REVIEW

File: src/constants/config.js [hunk 1]
```diff
@@ -5,13 +5,15 @@ const envConfig = {
     workflowUrl: 'https://pfe-preview.zooniverse.org/projects/rafe-dot-lafrance/notes-from-nature-on-staging/classify?reload=1&',
     panoptesReturnUrl: 'http://localhost:3000/',
     panoptesAppId: '24ad5676d5d25c6aa850dc5d5f63ec8c03dbc7ae113b6442b8571fce6c5b974c',
+    apiUrl: 'https://panoptes-staging.zooniverse.org/api/',
```
Comment (mcbouslog) [inline src/constants/config.js]: are the new `apiUrl` properties on the `staging` and `production` objects related to this PR or from unrelated work?

File: src/constants/expeditions.js [hunk 2]
```diff
@@ -346,6 +346,16 @@ export const expeditions = {
     image: 'Chenopodium_foliosum.jpg',
     link: 'http://bgbm.org/en/science/biological-collections',
   },
+  'Aquatics_Aquatic Insects of the Southeastern United States': {
```
Comment (mcbouslog) [inline src/constants/expeditions.js]: This was added in separate PR #175, will adding again create an issue, or maybe a rebase will fix?

**Comment by mcbouslog** [comment]
Closes #147

File: src/constants/expeditions.js (hunk 2 above)
Comment (rafelafrance) [reply src/constants/expeditions.js]: Another mess up on my part. I was juggling too many branches. We just need to make sure that we don't double up on the items.

File: src/constants/config.js (hunk 1 above)
Comment (rafelafrance) [reply src/constants/config.js]: No they were an attempt to bypass panoptes. They can go.

**Comment by rafelafrance** [comment]
Let me fix and redo the pull request.

**Comment by rafelafrance** [comment]
OK now?

**Review by mcbouslog — APPROVED** [review]

**Comment by mcbouslog** [comment]
yep, LGTM, thanks Rafe!

**Pull request merged**